readthedocs/core/management/commands/pull.py

```diff
@@ -20,6 +20,4 @@ def handle(self, *args, **options):
         if args:
             for slug in args:
                 version = utils.version_from_slug(slug, LATEST)
-                tasks.SyncRepositoryTask().run(
-                    version.pk,
-                )
+                tasks.sync_repository_task(version.pk)
```

Comment:
I assume that calling directly to run is the same as executing the task on the same host.

Reply:
Yeah, this should be equal